.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseURLExtractor.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Definitions.Chinese;
+using Microsoft.Recognizers.Text.Matcher;
+
+namespace Microsoft.Recognizers.Text.Sequence
+{
+    public class ChineseURLExtractor : BaseSequenceExtractor
+    {
+        public ChineseURLExtractor()
```

Comment:
The same principle of avoiding duplication should be applied here.
The base URL extractor could receive a config (like, for example HolidayParser) and the value inside the config would be used in the appropriate place.
English would use directly from the Base patterns, Chinese/Japanese from their CJ config (as far as I understand, Korean doesn't need this fix as they use whitespace as word separator).